ambry-named-mysql/src/main/java/com/github/ambry/named/MySqlNamedBlobDb.java

```diff
@@ -135,6 +135,30 @@ class MySqlNamedBlobDb implements NamedBlobDb {
   private static final String SOFT_DELETE_QUERY_V2 =
       String.format("UPDATE %s SET %s = ? WHERE %s", NAMED_BLOBS_V2, DELETED_TS, PK_MATCH_VERSION);
 
+  /**
+   * Pull the stale blobs that need to be cleaned up
+   */
+  private static final String GET_STALE_QUERY = String.format(""
```

Comment:
you probably have to turn off formatter with //@formatter:off and turn it back on after this statement.

Reply:
Sounds good, done.